lib/src/inline_parser.dart

```diff
@@ -27,12 +28,6 @@ class InlineParser {
     EmailAutolinkSyntax(),
     AutolinkSyntax(),
     LineBreakSyntax(),
-    // Allow any punctuation to be escaped.
-    EscapeSyntax(),
-    // "*" surrounded by spaces is left alone.
-    TextSyntax(r' \* ', startCharacter: $space),
-    // "_" surrounded by spaces is left alone.
-    TextSyntax(' _ ', startCharacter: $space),
```

Comment:
I guess these two `TextSyntax` intend to improve the performance? But the reality is opposite.

**With them:**

![Screenshot 2022-11-09 at 04 37 50](https://user-images.githubusercontent.com/5637609/200732496-01eb43f0-ef96-4b74-8ca9-8d95e000b4d7.png)

**Without them**

![Screenshot 2022-11-09 at 04 36 48](https://user-images.githubusercontent.com/5637609/200732572-45c86ecb-fe5f-4857-a8af-4f70e248ff01.png)

Reply:
Maybe I should not remove them in this PR, it is not related to the topic of this PR.

Reply:
Yeah, don't mix the "topics". A separate PR is good!